File: Clase4/EjercicioMenu.py
def Menu():
    print("Elige una opcion del menu \n 1. Personas \n 2. Vehiculos \n 3. Universidades \n 4. Notas \n 5. Salir \n")
    

Menu()
opcion = int(input("Elige una opcion del menu: \n"))
if opcion == 1:
    print ("Haz elegido la opcion del menu Personas")
    Menu()
elif opcion == 2:
    print ("Haz elegido la opcion del menu Vehiculos")
    Menu()
elif opcion == 3:
    print ("Haz elegido la opcion del menu Universidades")
    Menu()
elif opcion == 4:
    print ("Haz elegido la opcion del menu Notas")
    Menu()
elif opcion == 5:
    print ("Gracias por utilizar nuestra App. \n")
    SystemExit
else:
        print ("Elige una opcion del menu entre 1 - 5 \n")
        Menu()

File: Clase04/EjercicioMenu.java

import java.util.Scanner;

/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

/**
 *
 * @author Ing. Narvaez Mejia
 */
public class EjercicioMenu {

    public static void main(String[] args) {
        
        Scanner lectura = new Scanner (System.in);
        System.out.println("MENU PRINCIPAL: \n 1. PERSONAS.\n 2. VEHICULOS.\n 3. UNIVERSIDADES.\n 4. NOTAS.\n 5. SALIR");
        System.out.println("DIGITA UNA OPCION: ");
        int opcion = lectura.nextInt();
        
        switch(opcion){
        
            case 1: 
                System.out.println("SELECCIONASTE LA OPCION PERSONAS");
                break;
            case 2:
                System.out.println("SELECCIONASTE LA OPCION VEHICULOS");
                break;
            case 3:
                System.out.println("SELECCIONASTE LA OPCION UNIVERSIDADES");
                
                break;
            case 4:
                System.out.println("SELECCIONASTE LA OPCION NOTAS");
                break;
            case 5: 
                System.out.println("HASTA LUEGO");
                break;
            default:
                System.out.println("HAS DIGITADO UNA OPCION INVALIDA");
                break;
               
        }
        
     
    }
}
